src/containers/course-section/CourseSectionContainer.tsx

```diff
@@ -85,35 +217,55 @@ const CourseSectionContainer: FC<SectionProps> = ({
     {
       id: 1,
       label: (
-        <Box>{t('course.courseSection.resourcesMenu.lessonMenuItem')}</Box>
+        <Box sx={styles.menuItem}>
+          <SvgIcon
+            color='primary'
```

Comment:
can you move this to styles? If it isn`t working, try to import palette in styles and set {collor: palette.primary}